api/envoy/api/v2/rds.proto

```diff
@@ -44,7 +45,14 @@ service RouteDiscoveryService {
   }
 }
 
-// [#comment:next free field: 9]
+// [#protodoc-title: RouteConfiguration's virtual hosts updates]
+// * HTTP :ref:`router filter <config_http_filters_router>`
```

Comment:
Can you provide some discussion comparable to that for `RouteDiscoveryService`. In particular, should cover what the resource type is embedded in `Any` and how this related to RDS and expected use cases.

Reply:
Done.